test/utils.mjs

```diff
@@ -522,3 +522,9 @@ export const emptyL2 = async ({ nf3User, web3, logs }) => {
   }
   await new Promise(resolve => setTimeout(resolve, 6000));
 };
+
+export async function isTransactionMined(txHash, web3) {
```

Comment:
The name of this function is a bit confusing because it gives the impression that a boolean value is gonna be returned. Maybe changing it to `waitTransactionToBeMined` or something similar will match with the function's goal.

Reply:
True. Somehow I thought that given the recursive nature, the name might be okay. But I agree that using `wait` is better. Thank you.